langlib/lang.function/src/main/ballerina/function.bal

```diff
@@ -21,6 +21,23 @@ import ballerina/jballerina.java;
 # If the arguments specified in `args` are not of the type required by `func`,
 # then this will panic.
 #
+# ```ballerina
+# function fn = getGreeting;
+# // Call `getGreeting` without arguments.
+# any|error greeting = function:call(fn);
+#
+# // Call `getGreeting` with an argument.
```

Comment:
Shall we change this description to something like
`// Call `getGreeting` with name parameter.` wdyt?

Reply:
Did not go into detail here intentionally since at this point the parameter name(s) could be anything. As opposed to directly calling `getGreeting`.